.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "vendor/llhttp"]
     path = vendor/llhttp
     url = https://github.com/nodejs/llhttp.git
-    branch = v8.1.1
+    branch = v8.x
```

Comment:
We shouldn't have this unpinned, it'd make it impossible to track what version aiohttp is compiled against..

Reply:
Surely the pin is the commit of of the submodule. This is just the branch to update from, and there is no v8.1.1 branch.

Reply:
Hmm, actually, looks like we can do `tags/v8.1.1`.

Reply:
Nevermind, that doesn't work...